test/test_mps.py

```diff
@@ -10241,7 +10300,7 @@ def get_samples():
             self.assertEqual(cpu_out, mps_out, atol=atol, rtol=rtol)
 
 
-    @ops(mps_ops_grad_modifier(copy.deepcopy(op_db)), allowed_dtypes=MPS_GRAD_DTYPES)
+    @ops(mps_ops_grad_modifier(copy.deepcopy(test_consistency_op_db)), allowed_dtypes=MPS_GRAD_DTYPES)
```

Comment:
We are already doing a deep_copy where it's needed, have you seen any issues? Why do we need to copy it again.

Reply:
I left a comment here: https://github.com/pytorch/pytorch/pull/98167#discussion_r1155285682

In short, the op_db will be modified by the modifier which will cause unexpected xfail to other tests.

Reply:
test_output_match doesn't have op_db deep-copied.